build_push.sh

```diff
@@ -2,18 +2,16 @@
 set -e -x -u
 
 # required configuration:
-# * APP
-# * DOCKER_REPO
+# * DOCKER_SERVER (from ECR)
 
-tag="${1:-latest}"
+app=tile-server
+tag=$1
 
 # build docker image and tag it with git hash and aws environment
-docker build -t $APP:$tag .
-docker tag $APP:$tag $DOCKER_REPO:$tag
+docker build -t $app:$tag -t $app:latest .
```

Comment:
Will this error if someone omits the `tag` argument? Should it?

Reply:
I tried `docker build -t foo: .` to see what would happen if `$tag` is empty and it throws an error. I'm guessing that's probably what we want so that we don't end up with an image without any tags besides `latest`.